app/contracts/work_packages/base_contract.rb

```diff
@@ -326,8 +326,8 @@ def validate_people_visible(attribute, id_attribute, list)
 
       unless principal_visible?(id, list)
         errors.add attribute,
-                   I18n.t('api_v3.errors.validation.invalid_user_assigned_to_work_package',
-                          property: I18n.t("attributes.#{attribute}"))
+                   ": #{I18n.t('api_v3.errors.validation.invalid_user_assigned_to_work_package',
```

Comment:
I think the resulting error message looks a bit weird in english:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/83396/184940291-b255b283-3134-46e1-93ff-94c70561fb52.png">

Since we have the attribute name already included in the message, we can add the validation to the `:base` instead of the attribute. The error would look like this:

<img width="596" alt="image" src="https://user-images.githubusercontent.com/83396/184940091-184cee6d-5bcd-4ea1-b037-fcbecd910b96.png">

```
errors.add :base,
           I18n.t('api_v3.errors.validation.invalid_user_assigned_to_work_package',
                                       property: I18n.t("attributes.#{attribute}"))

Reply:
Thanks @dombesz, Yes I agree with you, it's not eye-catching in this way, and it looks better when there is no colon, at least in my eyes. But in the ticket it is written to emphasize on the affected field, we can add a colon after the attribute name. Only because of this I added the colon...

Reply:
If you change the errors from assigned to to base, it will not be properly mapped to the API and the frontend/clients will not be able to find out which field the error belonged to (which results in the field no longer being red)

If you want to change the format, you'll need to look into changing the format for errors I think: https://www.bigbinary.com/blog/rails-6-allows-to-override-the-activemodel-errors-full_message-format-at-the-model-level-and-at-the-attribute-level